main/templates/allauth/account/password_reset.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'main/base.html' %}
```

Comment:
customised the forgot_password.html made eariler to match allauth template in terms of fileds, id, etc.